https://github.com/xmrig/xmrig/pull/2882

--- ./ext/sse2neon.h.orig	2022-11-20 00:23:25.252409583 +0000
+++ ./ext/sse2neon.h	2022-11-20 00:23:30.429390486 +0000
@@ -390,7 +390,7 @@ FORCE_INLINE uint32_t _mm_crc32_u8(uint3
 
 // Older gcc does not define vld1q_u8_x4 type
 #if defined(__GNUC__) && !defined(__clang__) &&                        \
-    ((__GNUC__ <= 10 && defined(__arm__)) ||                           \
+    ((__GNUC__ <= 12 && defined(__arm__)) ||                           \
      (__GNUC__ == 10 && __GNUC_MINOR__ < 3 && defined(__aarch64__)) || \
      (__GNUC__ <= 9 && defined(__aarch64__)))
 FORCE_INLINE uint8x16x4_t _sse2neon_vld1q_u8_x4(const uint8_t *p)
