diff -uNrp a/app/codec/ffmpeg/ffmpegdecoder.h b/app/codec/ffmpeg/ffmpegdecoder.h
--- a/app/codec/ffmpeg/ffmpegdecoder.h	2022-02-28 05:47:17.000000000 +0000
+++ b/app/codec/ffmpeg/ffmpegdecoder.h	2022-08-09 13:12:07.305793787 +0100
@@ -29,6 +29,7 @@ extern "C" {
 #include <libavformat/avformat.h>
 #include <libswscale/swscale.h>
 #include <libswresample/swresample.h>
+#include <libavcodec/avcodec.h>
 }
 
 #include <QAtomicInt>
