--- olive-0.1.2/effects/internal/texteffect.cpp.orig	2021-01-01 12:02:53.808228887 +0100
+++ olive-0.1.2/effects/internal/texteffect.cpp	2021-01-01 12:03:29.551168827 +0100
@@ -25,6 +25,7 @@
 #include <QOpenGLTexture>
 #include <QTextEdit>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPushButton>
 #include <QColorDialog>
 #include <QFontDatabase>
--- olive-0.1.2/effects/internal/timecodeeffect.cpp.orig	2021-01-01 13:18:03.597706937 +0100
+++ olive-0.1.2/effects/internal/timecodeeffect.cpp	2021-01-01 13:18:38.725643649 +0100
@@ -25,6 +25,7 @@
 #include <QOpenGLTexture>
 #include <QTextEdit>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPushButton>
 #include <QColorDialog>
 #include <QFontDatabase>
--- olive-0.1.2/ui/graphview.cpp.orig	2021-01-01 13:26:15.729814018 +0100
+++ olive-0.1.2/ui/graphview.cpp	2021-01-01 13:27:10.035713140 +0100
@@ -21,6 +21,7 @@
 #include "graphview.h"
 
 #include <QPainter>
+#include <QPainterPath>
 #include <QMouseEvent>
 #include <QtMath>
 #include <cfloat>
--- olive-0.1.2/ui/timelineheader.cpp.orig	2021-01-01 13:35:17.228804660 +0100
+++ olive-0.1.2/ui/timelineheader.cpp	2021-01-01 13:36:05.999712798 +0100
@@ -21,6 +21,7 @@
 #include "timelineheader.h"
 
 #include <QPainter>
+#include <QPainterPath>
 #include <QMouseEvent>
 #include <QScrollBar>
 #include <QtMath>
